src/index.tsx

```diff
@@ -86,6 +88,26 @@ const startApp = async () => {
   const Root = () => {
     const alert = useAlert();
 
+    const { updateAvailable } = React .useContext(ServiceWorkerContext);
+
+    React.useEffect(() => {
+      if (updateAvailable) {
+        alert.show(
+          {
+            content: "Please refresh the page!",
+            title: "New version is available!"
```

Comment:
Maybe we should add a button that user could use straight away which would refresh the page?

Reply:
Usually, a hard refresh is required and some users don't know how to do that. It would be great if we could do the hard refresh for them.